src/polyfills.js

```diff
@@ -75,3 +75,58 @@ if (!String.prototype.padStart) {
     }
   };
 }
+
+// From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
+if (!Object.keys) {
```

Comment:
Why did you add keys? Keys is already supported natively by all modern browsers.

Reply:
Because I wasn't sure.... and the place that I got entries from had it as well.